src/Components/Web.JS/src/Platform/WebAssemblyResourceLoader.ts

```diff
@@ -5,7 +5,9 @@ const networkFetchCacheMode = 'no-cache';
 
 export class WebAssemblyResourceLoader {
   private usedCacheKeys: { [key: string]: boolean } = {};
+
```

Comment:
What rule is this?

Reply:
https://eslint.org/docs/rules/lines-between-class-members